cfme/tests/services/test_service_customer_bz.py

```diff
@@ -67,3 +71,46 @@ def test_edit_bundle_entry_point(appliance, provider, catalog_item, request):
                                                               "ServiceProvision_Template/"
                                                               "CatalogItemInitialization")
     view.cancel_button.click()
+
+
+@pytest.mark.tier(2)
+@pytest.mark.meta(automates=[BZ(1705021)])
+@pytest.mark.parametrize(
```

Comment:
Why is this parametrized? Planning on adding other files for this BZ case?

Reply:
To pass parameters to import_data fixture